test/parallel/test-repl-history-navigation.js

```diff
@@ -554,6 +555,36 @@ const tests = [
     expected: [],
     clean: false
   },
+  {
+    env: { NODE_REPL_HISTORY: defaultHistoryPath },
+    test: ['const util = {}', ENTER,
+           'ut', RIGHT, ENTER],
+    expected: [
+      prompt, ...'const util = {}',
+      'undefined\n',
+      prompt, ...'ut', ' // il', '\n// {}', 'il',
+      '\n// {}',
+      prompt,
+    ],
+    clean: false
+  },
+  {
+    env: { NODE_REPL_HISTORY: defaultHistoryPath },
+    test: [
+      'const utilDesc = Reflect.getOwnPropertyDescriptor(globalThis, "util")',
+      ENTER,
+      'globalThis.util = {}', ENTER,
+      'ut', RIGHT, ENTER,
+      'Reflect.defineProperty(globalThis, "util", utilDesc)', ENTER],
+    expected: [
+      prompt, ...'globalThis.util = {}',
+      '{}\n',
+      prompt, ...'ut', ' // il', 'il',
+      '{}\n',
```

Comment:
```suggestion
      prompt, ...'const utilDesc = ' +
                 'Reflect.getOwnPropertyDescriptor(globalThis, "util")',
      'undefined\n',
      prompt, ...'globalThis.util = {}',
      '{}\n',
      prompt, ...'ut', ' // il', 'il',
      '{}\n',
      prompt, ...'Reflect.defineProperty(globalThis, "util", utilDesc)',
      'true\n',
```

Reply:
Cool Thanks! do you know what the problem was?

Reply:
Oh right I was probably so out of focus now I see.